flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/model/node/NewConnectionBuilder.java

```diff
@@ -53,7 +53,7 @@ public class NewConnectionBuilder extends CallBuilder {
     public static final String CHECK_ERROR_DOC = "Terminate on error";
     public static final String CONNECTION_NAME_LABEL = "Connection Name";
     public static final String CONNECTION_TYPE_LABEL = "Connection Type";
-
+    public static final String CONNECTION_NAME_DOC = "Name of the connection instance";
```

Comment:
@hasithaa please review the new description of the `Connection Name` field

Reply:
+1 for "Name of the connection". Updated in https://github.com/ballerina-platform/ballerina-language-server/pull/39/commits/418c81fe88b522d9cf4c0d45a34da4512d25f6a3